src/modules/homeView.js

```diff
@@ -0,0 +1,144 @@
+          class="comments-button" 
+          data-rocket-id="${rocketId}" 
+          data-rocket-image="${rocketImage}"
+          data-rocket-name="${rocketName}"
+          data-rocket-type=${rocketType}
+          data-rocket-rate=${successRate}
+          data-rocket-cost=${costPerLaunch}
+          data-rocket-weight=${weightMass}
+        >
+        comments
+        </button>
+        </section>
+      </li>
+      `;
+    });
+    rocketsContainer.innerHTML = rockets;
+
+    // Add event listener to toggle heart icon color and track the icon button clicked
+    const likeIcons = document.querySelectorAll('.like-and-icon i');
+    likeIcons.forEach((icon) => {
+      icon.addEventListener('click', async () => {
+        const rocketId = icon.getAttribute('data-rocket-id');
+        icon.classList.toggle('liked');
+
+        try {
+          // initialize and add some likes to the likes count👍
+          await likeRocket(rocketId);
+        } catch (error) {
+          /* eslint-disable no-console */
+          console.log(error.message);
+          /* eslint-enable no-console */
+        }
+
+        // call the endpoint get the likes👍👍
+        try {
+          const data = await getLikes();
+          /* eslint-disable camelcase */
+          const updateLike = data.find(({ item_id }) => item_id === rocketId);
+          // update the like count of each rocket in the UI
+          if (updateLike) {
+            const likeCountElement = document.getElementById(`likeCount${rocketId}`);
+            likeCountElement.textContent = `${updateLike.likes} likes`;
+          } else {
+            const likeCountElement = document.getElementById(`likeCount${rocketId}`);
+            likeCountElement.textContent = '0 likes';
+          }
+        } catch (error) {
+          /* eslint-disable no-console */
+          console.log(error.message);
+          /* eslint-enable no-console */
+        }
+      });
+    });
+
+    // track the comment💬 button clicked
+    const commentButtons = document.querySelectorAll('.comments-button');
+
+    commentButtons.forEach((button) => {
+      button.addEventListener('click', async () => {
+        const rocketId = button.getAttribute('data-rocket-id');
+        const rocketImage = button.getAttribute('data-rocket-image');
+        const rocketName = button.getAttribute('data-rocket-name');
+        const successRate = button.getAttribute('data-rocket-rate');
+        const costPerLaunch = button.getAttribute('data-rocket-cost');
+        const weightMass = button.getAttribute('data-rocket-weight');
+        const rocketType = button.getAttribute('data-rocket-type');
+
+        // hide the homepage main contains
+        homeContainer.style.filter = 'blur(25px)';
+
+        // get comments💬💬 of this rockets🚀🚀
+        const commentData = await getComments(rocketId);
+
+        // console.log(commentData.length)
+        // localStorage.setItem('eachComment', JSON.stringify(commentData))
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧

Reply:
resolved